### ISSUE TITLE
Using ipfilter and secret

### ISSUE THREAD

**Issue by refda** [nextcloud/documentserver_community#232]
Hi, I just installed the ONLYOFFICE App and the documentserver_community. After this I was going to the ONLYOFFICE settings in my nextcloud instance and filling in the server address: https://www.xxx.xx/index.php/apps/documentserver_community/

Now everything works fine but I would like to restrict access to the documentserver via ipfilter and secret. How can I do this? There is no default.json inside /apps/documentserver_community/3rdparty/onlyoffice/documentserver...

And are my files still secure in my nextcloud instance?

tl;dr: I would like to secure both - the documentserver and my files but I could not find any information. Thanks!
